utils/zerovec/src/ule/encode.rs

```diff
@@ -110,6 +111,15 @@ unsafe impl<T: VarULE + ?Sized> EncodeAsVarULE<T> for &'_ T {
     }
 }
 
+unsafe impl<T: VarULE + ?Sized> EncodeAsVarULE<T> for Cow<'_, T>
+where
+    T: Clone,
```

Comment:
Actually, wait, I didn't notice this the first time, but it should be `T: ToOwned + ?Sized`, not `T: Clone`, because `str` is a valid `T` and `str` does not implement `Clone`.

Reply:
Dah, I know this, I just followed what rustc said instead of what I know to be correct 🙃

Reply:
Would you mind opening another PR to fix it?

Reply:
Will do

Reply:
See #2377